src/gui_common/dialogs/CustomConfirmationDialog.cs

```diff
@@ -25,6 +25,9 @@ public class CustomConfirmationDialog : CustomDialog
     [Signal]
     public delegate void Confirmed();
 
+    [Signal]
+    public delegate void Cancelled();
```

Comment:
Shouldn't this also be emitted when this is hidden?

Reply:
Ah yeah that's true, though confirming also hides the popup so this needs extra handling.

Reply:
That's correct, and code I've needed to write before to handle the case of hiding on success.

Reply:
Fixed.